shell/conf/benchmarks_defaults.conf

```diff
@@ -20,25 +20,13 @@
 # Logs
 ALOJA_FORCE_COLORS="1" #Force colors on screen even when writing log files
 
-# Data sizes and default scale factor
-if [ "$BENCH_SCALE_FACTOR" ] ; then
-   BENCH_DATA_SIZE="$(( BENCH_SCALE_FACTOR * 1000000000 ))" #in GB
-else
-   if [ ! "$BENCH_DATA_SIZE" ] ; then
-     # Check if TPC scale factor has been specified
-     if [[ "$TPCH_USE_LOCAL_FACTOR" > 0 ]]; then 
-       BENCH_DATA_SIZE="$(( TPCH_USE_LOCAL_FACTOR *  1000000000 ))" # if set, it is in GBs
-     else
-       if [ "$TPCH_SCALE_FACTOR" ] ; then
-         BENCH_DATA_SIZE="$(( TPCH_SCALE_FACTOR *  1000000000 ))" # if set, it is in GBs
-       else
-         BENCH_DATA_SIZE="100000000" #100MB, in bytes #1000000000000 1TB
-       fi
-     fi
-   fi
-fi
+BYTES_IN_MB=1000000
+BYTES_IN_GB=1000000000
+BYTES_IN_TB=1000000000000
```

Comment:
Do we actually use these anywhere any more? If not, I would just remove them and let the getters take care of the conversions.